projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py

```diff
@@ -0,0 +1,634 @@
+"""Base MODBUS functions.
+
+This module provide base functionalities to easy interact with:
+CN0414 - Analog input and HART for PLC/DCS systems
+CN0418 - Analog output and HART for PLC/DCS systems
+CN0416 - RS485 transceiver.
+"""
+
+__version__ = '0.1'
+__author__ = 'Mihai Ionut Suciu'
+__status__ = 'Development'
+
+from time import sleep
+from colorama import init, Fore
+import dcs_cn0435_utilities as utilities
+
+init(autoreset=True)
+
```

Comment:
nitpick: extra line

Reply:
Two lines are necessary to respect python linting.